### DISCUSSION TITLE
causer_type and causer_id saved with NULL

### DISCUSSION THREAD

**Discussion by mehdizadehrp** [spatie/laravel-activitylog#885] [Q&A]
I'm using version 1 of the package and in some records (not all) causer_type and causer_type saved with NULL, also I added user ip in description field and user ip saved as 127.0.0.1.
![ttas](https://user-images.githubusercontent.com/83898907/117921483-09c4cc00-b306-11eb-8eba-e67fd7434b3e.png)

What happening ?!

**Comment by Gummibeer** [comment]
You should upgrade everything until you can use at least v3 - better v4.
